test/boost/gossiping_property_file_snitch_test.cc

```diff
@@ -19,6 +19,15 @@
 
 static std::filesystem::path test_files_subdir("test/resource/snitch_property_files");
 
+namespace std {
+
+std::ostream& boost_test_print_type(std::ostream& os, const std::pair<sstring, sstring>& p) {
+    fmt::print(os, "{}:{}", p.first, p.second);
+    return os;
+}
+
+}
```

Comment:
When the migration process is complete, we can have a boost_test_print_type() that forwards everything to fmt.